aim/web/ui_v2/src/types/utils/d3/drawParallelHoverAttributes.d.ts

```diff
@@ -1,6 +1,7 @@
 import { IGetAxisScale } from './getAxisScale';
 import { ILineDataType, ILinesDataType } from './drawParallelLines';
 import { DimensionsType } from './drawParallelAxes';
+import { ISyncHoverStateParams } from './drawHoverAttributes';
 import React from 'react';
 import HighlightEnum from '../../../components/HighlightModesPopover/HighlightEnum';
```

Comment:
You can also import with absolute path like 'components/HighlightModesPopover/HighlightEnum'  to avoid long path name